sample_tracking/app/i18n/locales/en/translations.json

```diff
@@ -81,7 +81,7 @@
   "download": "Download",
   "successfullyImportedFile" : "Successfully imported file",
   "great": "Great!",
-  "originValueError": "origin value should be one of the following: known, uncertain, or unknown",
+  "originValueError": "trusted value should be one of the following: known, uncertain, or unknown",
```

Comment:
there seem to be a lot of uses of 'origin'  should you do a full search replace?  This doesnt seem related to this change.

Reply:
yeah I think a followup bug fix that would be a bit more invasive would be replacing all references of origin with trusted.